arduino/smartcar/smartcar.ino

```diff
@@ -1,9 +1,34 @@
 #include <Smartcar.h>
 
+
+const int cruiseSpeed = 40;
+const int triggerDist = 200;
+
+const int lDegrees = -75; // degrees to turn left
+
+float distance;
```

Comment:
distance is an `int` not a float and you probably don't need a global variable for it